package.json

```diff
@@ -63,7 +63,8 @@
     "@angular/forms": "^8.1.2",
     "@angular/material": "^8.1.1",
     "@angular/router": "^8.1.2",
-    "firebase": "^6.3.1"
+    "firebase": "^6.3.1",
+    "lodash": "^4.17.15"
```

Comment:
why did you move lodash from dev dependencies to normal dependencies ?

Reply:
Yes, that's not necessary anymore. My bad, I forgot to revert that. I was using it at runtime first but removed it later. it can stay in devDependencies now.